critiquebrainz/frontend/views/bb_author.py

```diff
@@ -0,0 +1,105 @@
+from datetime import datetime
+from flask import Blueprint, render_template, request
+from flask_login import current_user
+from werkzeug.exceptions import NotFound, BadRequest
+from flask_babel import gettext
+import critiquebrainz.db.review as db_review
+import critiquebrainz.frontend.external.bookbrainz_db.author as bb_author
+import critiquebrainz.frontend.external.bookbrainz_db.literary_work as bb_literary_work
+from critiquebrainz.frontend.forms.rate import RatingEditForm
+from critiquebrainz.frontend.views import get_avg_rating, AUTHOR_REVIEWS_LIMIT
+
+bb_author_bp = Blueprint('bb_author', __name__)
+
+LITERARY_WORK_TYPE_NOVEL = 'Novel'
+LITERARY_WORK_TYPE_SHORT_STORY = 'Short Story'
+LITERARY_WORK_TYPE_POEM = 'Poem'
+LITERARY_WORK_TYPE_OTHER = 'other'
+VALID_LITERARY_WORK_PARAMS = [LITERARY_WORK_TYPE_NOVEL, LITERARY_WORK_TYPE_SHORT_STORY, LITERARY_WORK_TYPE_POEM, LITERARY_WORK_TYPE_OTHER]
+
+@bb_author_bp.route('/<uuid:id>')
+def entity(id):
```

Comment:
This file seems to be indented with tabs, instead of 4 spaces

Reply:
Fixed

Reply:
Fixed